recipe-page.html

```diff
@@ -0,0 +1,85 @@
+        </section>
+      </header>
+      <main>
+        <h1> Chocolate Pizza </h1>
+        <section class='mainSubHeader'>
+          <p> POSTED ON 15 DEC 2013 / DESSERTS </p>
+          <section class='rightAlignedMainSubHeader'>
+            <img src='challenge/print-icon.png' />
+            <p> PRINT </p>
+          </section>
+        </section>
+        <img src='challenge/choco-pizza.png' />
+        <section class='recipeInstructions' />
+          <p> For the fig-swirl: Melt the butter over medium heat in a saucepan.  Add brown sugar and   stir to dissolve.  Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. </p>
+          <p> Ice cream: In a small pot over medium heat, combine milk and granulated sugar until   sugar is completely dissolved and the milk is just barely lukewarm.  Whisk in the egg yolks.  Set mixture in the fridge and wait until the fig mixture is cooled.  </p>
+          <p>  Using an ice cream machine, pour liquids into the frozen basin and process according to  the manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding   mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes.  Pour semi -frozen mixture into a pyrex dish or glass tupperware.  Freeze for at least 2 hours before   serving. </p>
+        </section>
+        <section class='ingredientsList'>
+          <ul>
+            <section>
+              <li><input type='checkbox'> 1 1/2 cups milk</li>
+              <li><input type='checkbox'> 1/2 cup mascarpone</li>
+              <li><input type='checkbox'> 1/2 tsp pink salt</li>
+              <li><input type='checkbox'> 1lb Black Mission Figs</li>
+              <li><input type='checkbox'> 1/2 cup brown sugar</li>
+              <li><input type='checkbox'> 2-4 tbsp water</li>
+            </section>
+            <section class='secondIngredientsSection'>
+              <li><input type='checkbox'> 1 1/2 cups heavy cream</li>
+              <li class="checked"><input type='checkbox' checked> 1/3 cup granulated sugar</li>
+              <li class="checked"><input type='checkbox' checked> 2 egg yolks</li>
+              <li><input type='checkbox'> 1 lemon, juiced</li>
+              <li><input type='checkbox'> 2 tbsp butter</li>
+              <li><input type='checkbox'> 1 cup honey roasted pecans, roughly chopped</li>
+            </section>
+          </ul>
+        </section>
+      </main>
+      <footer>
+        <img src='challenge/van-pic.png' />
+        <section class='headerTextSection'>
+          <h1>Vanessa Stevenson</h1>
+          <p>Food enthusiast, photography fan.  Add a pinch of raw foodism and that's pretty much who   I am.</p>
+        </section>
+        <section class='shareRecipe'>
+          <p> SHARE RECIPE </p>
+        </section>
+        <section class='endOfPage'>
+          <br />
```

Comment:
We don't use the br tag any more. The reason being that it's trying to us html to adjust presentation. Use css to achieve that visual space.